packages/shared/src/proj/projection.tile.matrix.set.ts

```diff
@@ -59,11 +53,8 @@ export class ProjectionTileMatrixSet {
      * @param epsgCode
```

Comment:
Do we still need this?
`static altMap = new Map<EpsgCode, Map<string, ProjectionTileMatrixSet>>();`

Reply:
You are right we should delete this, however it gets deleted when I remove `ProjectionTileMatrixSet` in the next PR :)